src/components/common/Layout/components/NavMenu/NavMenu.tsx

```diff
@@ -96,7 +96,7 @@ const NavMenu = (): JSX.Element => {
                 rel="noopener"
                 leftIcon={<XLogo />}
               >
-                Twitter
+                X
```

Comment:
Let's keep it "Twitter" here for now, X (next to another X) looks really strange

Reply:
yeah, looks strange...

Reply:
but in this case every place I changed will look strange